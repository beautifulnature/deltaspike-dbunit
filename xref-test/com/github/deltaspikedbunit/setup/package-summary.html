
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Deltaspike DBUnit 0.5.0 Reference Package com.github.deltaspikedbunit.setup</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>
		      	<div class="overview">
        	<ul>
          		<li>
            		<a href="../../../../overview-summary.html">Overview</a>
          		</li>
          		<li class="selected">Package</li>
        	</ul>
      	</div>
      	<div class="framenoframe">
        	<ul>
          		<li>
            		<a href="../../../../index.html" target="_top">FRAMES</a>
          		</li>
          		<li>
            		<a href="package-summary.html" target="_top">NO FRAMES</a>
          		</li>
        	</ul>
      	</div>
		
		      	<h2>Package com.github.deltaspikedbunit.setup</h2>

		<table class="summary">
        	<thead>
          		<tr>
            		<th>Class Summary</th>
          		</tr>
        	</thead>
        	<tbody>
        		            	<tr>
              		<td>
                		<a href="CleanInsertSetupOnClassTest.html" target="classFrame">CleanInsertSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="CleanInsertSetupOnMethodTest.html" target="classFrame">CleanInsertSetupOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteAllSetupOnClassTest.html" target="classFrame">DeleteAllSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteAllSetupOnMethodTest.html" target="classFrame">DeleteAllSetupOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteSetupOnClassTest.html" target="classFrame">DeleteSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteSetupOnMethodTest.html" target="classFrame">DeleteSetupOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="InsertSetupOnClassTest.html" target="classFrame">InsertSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="InsertSetupOnMethodTest.html" target="classFrame">InsertSetupOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="MixedSetupOnClassAndMethodTest.html" target="classFrame">MixedSetupOnClassAndMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="MultipleInsertSetupOnClassTest.html" target="classFrame">MultipleInsertSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="RefreshSetupOnClassTest.html" target="classFrame">RefreshSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="RefreshSetupOnMethodTest.html" target="classFrame">RefreshSetupOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="TruncateSetupOnClassTest.html" target="classFrame">TruncateSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="TruncateSetupOnMethodTest.html" target="classFrame">TruncateSetupOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="UpdateSetupOnClassTest.html" target="classFrame">UpdateSetupOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="UpdateSetupOnMethodTest.html" target="classFrame">UpdateSetupOnMethodTest</a>
              		</td>
            	</tr>
				        	</tbody>
      	</table>
		
		      	<div class="overview">
        	<ul>
          		<li>
            		<a href="../../../../overview-summary.html">Overview</a>
          		</li>
          		<li class="selected">Package</li>
        	</ul>
      	</div>
      	<div class="framenoframe">
        	<ul>
          		<li>
            		<a href="../../../../index.html" target="_top">FRAMES</a>
          		</li>
          		<li>
            		<a href="package-summary.html" target="_top">NO FRAMES</a>
          		</li>
        	</ul>
      	</div>
				<hr />
		Copyright &#169; 2015. All Rights Reserved.
	</body>
</html>