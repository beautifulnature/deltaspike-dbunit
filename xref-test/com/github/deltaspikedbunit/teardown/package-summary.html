
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Deltaspike DBUnit 0.5.0 Reference Package com.github.deltaspikedbunit.teardown</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>
		      	<div class="overview">
        	<ul>
          		<li>
            		<a href="../../../../overview-summary.html">Overview</a>
          		</li>
          		<li class="selected">Package</li>
        	</ul>
      	</div>
      	<div class="framenoframe">
        	<ul>
          		<li>
            		<a href="../../../../index.html" target="_top">FRAMES</a>
          		</li>
          		<li>
            		<a href="package-summary.html" target="_top">NO FRAMES</a>
          		</li>
        	</ul>
      	</div>
		
		      	<h2>Package com.github.deltaspikedbunit.teardown</h2>

		<table class="summary">
        	<thead>
          		<tr>
            		<th>Class Summary</th>
          		</tr>
        	</thead>
        	<tbody>
        		            	<tr>
              		<td>
                		<a href="CleanInsertTearDownOnClassTest.html" target="classFrame">CleanInsertTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="CleanInsertTearDownOnMethodTest.html" target="classFrame">CleanInsertTearDownOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteAllTearDownOnClassTest.html" target="classFrame">DeleteAllTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteAllTearDownOnMethodTest.html" target="classFrame">DeleteAllTearDownOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteTearDownOnClassTest.html" target="classFrame">DeleteTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="DeleteTearDownOnMethodTest.html" target="classFrame">DeleteTearDownOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="InsertTearDownOnClassTest.html" target="classFrame">InsertTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="InsertTearDownOnMethodTest.html" target="classFrame">InsertTearDownOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="MixedTearDownOnClassAndMethodTest.html" target="classFrame">MixedTearDownOnClassAndMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="MultipleInsertTearDownOnClassTest.html" target="classFrame">MultipleInsertTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="RefreshTearDownOnClassTest.html" target="classFrame">RefreshTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="RefreshTearDownOnMethodTest.html" target="classFrame">RefreshTearDownOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="TruncateTearDownOnClassTest.html" target="classFrame">TruncateTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="TruncateTearDownOnMethodTest.html" target="classFrame">TruncateTearDownOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="UpdateTearDownOnClassTest.html" target="classFrame">UpdateTearDownOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="UpdateTearDownOnMethodTest.html" target="classFrame">UpdateTearDownOnMethodTest</a>
              		</td>
            	</tr>
				        	</tbody>
      	</table>
		
		      	<div class="overview">
        	<ul>
          		<li>
            		<a href="../../../../overview-summary.html">Overview</a>
          		</li>
          		<li class="selected">Package</li>
        	</ul>
      	</div>
      	<div class="framenoframe">
        	<ul>
          		<li>
            		<a href="../../../../index.html" target="_top">FRAMES</a>
          		</li>
          		<li>
            		<a href="package-summary.html" target="_top">NO FRAMES</a>
          		</li>
        	</ul>
      	</div>
				<hr />
		Copyright &#169; 2015. All Rights Reserved.
	</body>
</html>