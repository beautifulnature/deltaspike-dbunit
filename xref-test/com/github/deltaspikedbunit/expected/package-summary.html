
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Deltaspike DBUnit 0.5.0 Reference Package com.github.deltaspikedbunit.expected</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>
		      	<div class="overview">
        	<ul>
          		<li>
            		<a href="../../../../overview-summary.html">Overview</a>
          		</li>
          		<li class="selected">Package</li>
        	</ul>
      	</div>
      	<div class="framenoframe">
        	<ul>
          		<li>
            		<a href="../../../../index.html" target="_top">FRAMES</a>
          		</li>
          		<li>
            		<a href="package-summary.html" target="_top">NO FRAMES</a>
          		</li>
        	</ul>
      	</div>
		
		      	<h2>Package com.github.deltaspikedbunit.expected</h2>

		<table class="summary">
        	<thead>
          		<tr>
            		<th>Class Summary</th>
          		</tr>
        	</thead>
        	<tbody>
        		            	<tr>
              		<td>
                		<a href="ExpectedFailureOnClassTest.html" target="classFrame">ExpectedFailureOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedFailureOnMethodTest.html" target="classFrame">ExpectedFailureOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedNonStrictOnClassTest.html" target="classFrame">ExpectedNonStrictOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedNonStrictOnMethodTest.html" target="classFrame">ExpectedNonStrictOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedNonStrictUnorderedOnClassTest.html" target="classFrame">ExpectedNonStrictUnorderedOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedNonStrictUnorderedOnMethodTest.html" target="classFrame">ExpectedNonStrictUnorderedOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedOnClassAndMethodTest.html" target="classFrame">ExpectedOnClassAndMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedOnClassAndMethodWithoutOverrideTest.html" target="classFrame">ExpectedOnClassAndMethodWithoutOverrideTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedQueryNonStrictOnMethodTest.html" target="classFrame">ExpectedQueryNonStrictOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedQueryOnMethodTest.html" target="classFrame">ExpectedQueryOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">ExpectedQueryWithModifierOnMethodTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedTableNonStrictOnClassTest.html" target="classFrame">ExpectedTableNonStrictOnClassTest</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">InnerModifier</a>
              		</td>
            	</tr>
				            	<tr>
              		<td>
                		<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">OuterModifier</a>
              		</td>
            	</tr>
				        	</tbody>
      	</table>
		
		      	<div class="overview">
        	<ul>
          		<li>
            		<a href="../../../../overview-summary.html">Overview</a>
          		</li>
          		<li class="selected">Package</li>
        	</ul>
      	</div>
      	<div class="framenoframe">
        	<ul>
          		<li>
            		<a href="../../../../index.html" target="_top">FRAMES</a>
          		</li>
          		<li>
            		<a href="package-summary.html" target="_top">NO FRAMES</a>
          		</li>
        	</ul>
      	</div>
				<hr />
		Copyright &#169; 2015. All Rights Reserved.
	</body>
</html>